commons/commands.py

```diff
@@ -639,7 +639,7 @@
 
 # LC Support Bundle
 SUPPORT_BUNDLE_LC = "/opt/seagate/cortx/utils/bin/cortx_support_bundle generate " \
-                    "-c yaml:///etc/cortx/cluster.conf -t {} -b {} -m \"{}\""
+                    "-c consul://cortx-consul-server:8500/conf -t {} -b {} -m \"{}\""
```

Comment:
is this port number constant or will change with every deployment?

Reply:
this will be constant, will not change with deployment.
dev team may change the port but not with deployment

Reply:
ok got it